utils/check-style/codespell-ignore-words.list

```diff
@@ -1,7 +1,14 @@
-thenn
+wthenn
 fpr
 creat
 parsering
 nd
 ect
 pullrequest
+pullrequests
+ans
+splitted
```

Comment:
splitted -> split? or there is something more complicated?

Reply:
fixed